modules/node_groups/node_groups.tf

```
@@ -69,6 +69,12 @@ resource "aws_eks_node_group" "workers" {
     }
   }
 
+  timeouts {
+    create = lookup(each.value["timeouts"], "create", "60m")
```

for the fallback on all of these, can we use `null` so that we use whats provided by the upstream provider (looks like it defaults to an hour https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/eks_node_group#timeouts )

```suggestion
    create = lookup(each.value["timeouts"], "create", null)
    update = lookup(each.value["timeouts"], "update", null)
    delete = lookup(each.value["timeouts"], "delete", null)
```

---

I'll try that. When I was doing my testing it didn't actually fallback and created empty values that broke the plan. But I will try again.